management/mailconfig.py

```diff
@@ -102,6 +102,18 @@ def get_mail_users(env):
 	users = [ row[0] for row in c.fetchall() ]
 	return utils.sort_email_addresses(users, env)
 
+def sizeof_fmt(num):
```

Comment:
I think this was on the forked branch but not in the original PR -- is this something you'd want me to change?